src/pixelator/plot/__init__.py

```diff
@@ -545,9 +561,9 @@ def plot_3d_graph(
     pxl_data: PixelDataset,
     component: str,
     marker: Union[list, None] = None,
-    layout_algorithm: Literal[
-        "fruchterman_reingold_3d", "kamada_kawai_3d", "pmds_3d"
-    ] = "fruchterman_reingold_3d",
+    layout_algorithm: Union[
+        Literal["fruchterman_reingold_3d", "kamada_kawai_3d", "pmds_3d"], None
+    ] = None,
```

Comment:
I would just factor this out into a LayoutAlgorithms alias and then use `LayoutAlgorithms | None` here.
(Since we dropped python <3.10 support we can use the new union syntax).

The same goes for the input to `plot_2d_graph`.